### DISCUSSION TITLE
Sign  Up

### DISCUSSION THREAD

**Discussion by JeimonkAs** [cssbattle/forum#303] [Q&A]
Hello,i return to css battle , and try do login , if i use github they say 
![image](https://user-images.githubusercontent.com/93387877/188266187-9857ea71-55c9-422d-8af4-3a6adcc9511b.png)
im trying send sign in email  but i cant got it :C

**Comment by chinchang** [comment]
Please DM us your email address or username on Twitter or discord (links in
website footer)

On Sat, 3 Sep 2022 at 3:49 PM, Pavel ***@***.***> wrote:

> Hello,i return to css battle , and try do login , if i use github they say
> [image: image]
> <https://user-images.githubusercontent.com/93387877/188266187-9857ea71-55c9-422d-8af4-3a6adcc9511b.png>
> im trying send sign in email but i cant got it :C
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cssbattle/forum/discussions/303>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAC4YDQ4NKADCY62SJA7PHTV4MQZ3ANCNFSM6AAAAAAQD24KFM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
*Kushagra Gour*
kushagra.dev
@chinchang457 <https://twitter.com/chinchang457>

**Reply by JeimonkAs** [reply]
> Please DM us your email address or username on Twitter or discord (links in website footer)
> On Sat, 3 Sep 2022 at 3:49 PM, Pavel ***@***.***> wrote: Hello,i return to css battle , and try do login , if i use github they say [image: image] <https://user-images.githubusercontent.com/93387877/188266187-9857ea71-55c9-422d-8af4-3a6adcc9511b.png> im trying send sign in email but i cant got it :C — Reply to this email directly, view it on GitHub <#303>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAC4YDQ4NKADCY62SJA7PHTV4MQZ3ANCNFSM6AAAAAAQD24KFM> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
> -- *Kushagra Gour* kushagra.dev @chinchang457 <https://twitter.com/chinchang457>

freewaterfall@mail.ru its my email
